fe/fe-core/src/test/java/org/apache/doris/analysis/AlterTableStmtTest.java

```diff
@@ -64,34 +63,6 @@ public void setUp() {
         };
     }
 
-    @Test
-    public void testNormal() throws UserException {
```

Comment:
why remove these tests and not add new case?

Reply:
recovered